.eslintrc.json

```diff
@@ -6,15 +6,22 @@
     "airbnb-base/legacy"
   ],
   "globals": {
+    "afterEach": true,
+    "beforeEach": true,
+    "afterAll": true,
+    "beforeAll": true,
     "describe": true,
+    "expect": true,
     "it": true,
     "fit": true,
-    "expect": true,
-    "xit": true,
+    "test": true,
     "xdescribe": true,
-    "fdescribe": true
-  }, 
+    "fdescribe": true,
+    "xit": true,
+    "xtest": true
+  },
   "rules": {
+    "linebreak-style": "off",
```

Comment:
I will upload a `.gitattributes` file so that won't be an issue for you anymore. I'm using `autocrlf = input`.